BENG183\_Project
================
Kelsey Dang, Katy Zou, Diego Quintero
12/9/2019

# Precision Medicine - Past, Present, & Future

## Table of Contents:

1.  Introduction  
2.  History of Precision Medicine with focus on Breast Cancer  
3.  Genomics in Precision Medicine  
4.  Techniques/Methods  
5.  Benefits of Precision Medicine  
6.  Drawbacks of Precision Medicine  
7.  Future Directions  
8.  Conclusion  
9.  References  
10. Supplementary Material

## 1\. Introduction

If every individual on the planet has their own unique genetic makeup,
one that distinguishes them from every other person, why are they seen
as the same when treating diseases? Each treatment applies differently
for each individual, it may or may not be effective, or might not have
the same effects on every patient. So how is this problem approached?
With Precision Medicine\!  

According to the National Research Council’s Toward Precision Medicine,
precision medicine is defined as, “the tailoring of medical treatment to
the individual characteristics of each patient”.<sup>1</sup> Precision
Medicine (PM), is being able to improve the treatment for patients,
while reducing the need for unnecessary diagnostic testing, therapies,
and misdiagnoses. It is a philosophy of medicine that treats patients
based on an understanding of their genetics, instead of “assigning”
patients to a treatment, a treatment is made catering to the patient.  

The goal of PM is not only to develop a health care system in which
research is carried out in the context of clinical care, but also it is
optimizing the treatment and knowledge used to deliver improved patient
outcomes.<sup>1</sup> By incorporating patient data, i.g. DNA, RNA,
proteins, metabolism, epigenome, and environmental aspects, Precision
Medicine can utilize these essential factors into techniques such as,
patient subtyping, biomarker discovery, drug repurposing, and
personalized treatment.  

Today, a large number of precision medicine techniques are implemented
to healthcare to help individuals before birth, at birth, and throughout
their lifetime. For example, when an expecting mother is about 8 – 12
weeks pregnant, she may have genetic testing to assess any chromosomal
abnormalities of the fetus, and the fetus’ entire genome may even be
sequenced. At birth, sequencing can be utilized in rapidly diagnosing
any factor that may lead to reduced mortality. Throughout an
individual’s life, these precision medicine techniques can be used in
the diagnosis and treatment of a variety of diseases, especially in
accurately diagnosing cancer and applying the correct therapy for
chronic diseases.<sup>1</sup>

## 2\. History of Precision Medicine with focus on Breast Cancer

![timeline](https://github.com/kelseydang/BENG183_Project/blob/master/BENG183_Project/timeline%20precision%20medicine.png)

**1882:** William Halsted performed the first radical mastectomy, a
procedure that surgically removes the breast, underlying chest muscle
(including pectoralis major and pectoralis minor), and lymph nodes of
the axilla.3 This surgery remained the standard breast cancer treatment
well into the 20th century.<sup>2</sup>

**1895:** The first X-ray was taken. Emil Herman Grubbe used X-rays to
treat a breast cancer patient.<sup>4</sup>

**1898:** Marie and Pierre Curie discovered the radioactive elements
radium and polonium. Radium was used for radiation therapy on patients
but its penetrating abilities were not well understood.<sup>2</sup>

**1932:** A new, less invasive mastectomy approach was
developed.<sup>2</sup>

**1937:** Radiation therapy is used in conjunction with surgery in order
to preserve the breast. The tumor is removed and needles laced with
radium are inserted near the breast and lymph nodes.<sup>2</sup>

**1978:** The anti-estrogen drug tamoxifen is approved by the Food and
Drug Administration (FDA) for use in breast cancer treatment. Tamoxifen
is part of a new class of drugs called selective estrogen receptor
modulators (SERMs) used against cancer. Each cell types’ unique receptor
composition allows for selective growth inhibition.5 This is a huge
breakthrough in understanding gene and protein interaction pathways.

**1984:** Researchers discovered the human equivalent gene HER2 in rats.
HER2 was aggressive when overexpressed and nonresponsive to treatments,
posing a serious dilemma for researchers.<sup>2</sup>

**1986:** Scientists clone the HER2 gene.

**1995:** Using cloning technology, scientists can now clone the tumor
suppressor genes BRCA1 and BRCA2. Inherited mutations in these two genes
predict an increased risk of breast cancer.

**1996:** The FDA approves anastrozole as an anti-estrogen treatment for
breast cancer.

**1998:** Tamoxifen, another SERM, is found to decrease the risk of
developing breast cancer in at-risk men and women by 50%. It is now the
most commonly prescribed preventative therapy for
“hormone-receptor-positive, early-stage breast cancer after surgery”.6
Trastuzumab, better known as Herceptin, is approved by the FDA. It is
used to target patients with HER2 receptor positive breast
cancer.<sup>7</sup>

**April 2003:** Human Genome Project is completed. This allowed the
possibility of precision medicine to be widely accepted.

**2006:** The drug raloxifene was found to reduce breast cancer risk for
postmenopausal women who have higher risk.

**2015:** All of US initiated was founded under the Obama administration
in efforts to expand the database for precision medicine focused
research.<sup>1</sup>

## 3\. Genomics in Precision Medicine

In the first of the charts below, the most widely prescribed breast
cancer drugs are listed along with the specific patient type targeted.
The second chart lists important biomarkers associated with breast
cancer, a description of what the biomarker is, the patient type it
affects, and the modern treatment prescribed.

![drug table](https://github.com/kelseydang/BENG183_Project/blob/master/BENG183_Project/drug%20table.png)

![biomarker table](https://github.com/kelseydang/BENG183_Project/blob/master/BENG183_Project/biomarker%20table.png)

Genomics is a branch of biology that focuses on understanding genomes
and comprises of the central core of precision medicine.<sup>24</sup>
Given the knowledge that every individual’s genetic makeup differs,
biomarkers are used in order to identify commonalities between carriers
of diseases. In this section, we will focus on genomic biomarkers that
identify breast cancer patient types, specifically delving into
HER2-positive patients who have an overexpression of the HER2 gene.  

The discovery of the HER2 gene pathway and its importance in breast
cancer growth led to the development of the drug trastuzumab and various
other targeted treatments, improving the survival rate of breast cancer
patients. The HER2 gene produces proteins that when overexpressed leads
to uncontrolled proliferation of epidermal tissue. The two NCI-funded
research teams led separately by Dennis Slamon, M.D. and Stuart
Aaronson, M.D. were able to discover the “groundbreaking hypothesis that
If HER2 could be blocked, the growth of HER2-positive breast cancer
might be slowed.”<sup>25</sup> This was done by showing that high levels
of HER2 protein correlate with faster breast cancer growth. A solution
they found was to inhibit the actions of the HER2 proteins with special,
laboratory produced monoclonal antibodies. So thus, the drug trastuzumab
or Herceptin was created.

## 4\. Techniques/Methods

Precision Medicine can provide a personalized insight into drug
treatment and disease diagnosis by using machine learning(ML) and
network-based (NB) methods. The issue here, however, is that due to the
vast range of biological factors at play in a person’s physiology, it is
impossible to create such a diagnosis based solely on just one of these
biological markers. The way precision medicine is starting to shift is
to using integrative analysis that utilizes two or more of a person’s
biological markers simultaneously to create a better prediction.

Common methods in machine learning fall under three categories:

![Common Methods in Machine Learning](https://github.com/kelseydang/BENG183_Project/blob/master/BENG183_Project/ML_methods.png)


  - Supervised: methods are commonly used for classification and
    regression and require known labels as input with your data.
    Supervised learning usually involves using a subset of the labeled
    data as a training set and the remaining of the data as the testing
    set to determine its accuracy.  
  - Unsupervised: methods are commonly used for clustering and
    dimensionality reduction and take as input a dataset that is
    unlabeled. For unsupervised learning, hidden patterns can be
    organized into meaningful subsets.  
  - Semisupervised: learning methods takes a mixture of both labeled and
    unlabeled samples that can be used to explain the structure of the
    data as well as make new predictions of unlabeled samples.

Example focused on Supervised Learning:

**Integrating both NB and ML approaches using Proteomics/Interactomics
and Transcriptomics and a support vector machine(SVM) to predict Drug
response**

Affinity-purification Mass Spectrometry(AP-MS) is used to determine the
physical interactions(preys) for a selected protein of interest (Bait)
in a sample; such interactions are known as protein-protein
interaction(PPIs). Selecting a target Bait that we know to be associated
with a specific disease(i.e. cancer) can lead us to other interactors
that could be essential for the development of the cancer phenotype.
After running AP-MS and determining which subset of preys have a strong
interaction, we can look at that subset from publicly available mRNA
expression data from patients who also have data of their response to
cancer treatment drug. Since we are given the drug responses of the
patients (1 for positive response, 0 otherwise), then we can use the
mRNA expression data of \~80% of the patients to train the SVM while
also inputting how those 80% of patients responded to the drug.

![training SVM](https://github.com/kelseydang/BENG183_Project/blob/master/BENG183_Project/TrainingSVM.jpg)

This trains the SVM to try and find a correlation of expression levels
for that subset of preys that had a strong interaction with your
original bait and how that patient responded. We can test the SVM using
the remaining 20% of patient mRNA data into the SVM and comparing how
accurate the predicted response form the SVM is with the actual response
from the given patient data.  

![testing SVM](https://github.com/kelseydang/BENG183_Project/blob/master/BENG183_Project/Testing%20SVM.jpg)

**How a SVM classifies data points: **

#### Linearly separable data:

The SVM is trained with an original set of labeled data and a hyperplane
is created to separate the data and positioned in space so that the
distance between the hyperplane and the closest data points from each
distinct label is maximized. (See figures below) This hyperplane is then
used as a boundary to determine where new data lies and how it is
classified.

![data1](https://github.com/kelseydang/BENG183_Project/blob/master/BENG183_Project/SVM_data1.png)

User input data w/ labels(above)

![data2](https://github.com/kelseydang/BENG183_Project/blob/master/BENG183_Project/SVM_data2.png)

Data points separated by Hyperplane with optimal Maximized Support Vector distances(above)

#### Non-linearly separable data:

Data points aren’t always able to be separated by just a linear line, so in order to account for this the SVM converts the data into a higher dimension where the data is able to be linearly separated by a hyperplane(purple line). Figures below demonstrate how data points can become linearly separable by first being converted to a higher dimension.

![nonlinear](https://github.com/kelseydang/BENG183_Project/blob/master/BENG183_Project/SVM_NonLinear.png)

Demo of SVM:
https://cs.stanford.edu/people/karpathy/svmjs/demo/

Optinoal exercise using R:
http://uc-r.github.io/svm

Example method using Unsupervised Learning:

K-means algorithm is a way of dividing points(genes, samples, cells, etc.) into homogeneous classes or clusters. It analyses data and groups them into a user-defined number of classes(clusters) based on similarity. This method takes in unlabeled data to begin with, the number of clusters desired, and can also take the number of maximum iterations.

#### K-means Algorithm

Start with suitable choices of number of clusters (K). Then:

    1. Assign each points to similar center
    2. Identify the cluster centroid
    3. Reassign the points based on the minimum distance to the new centroid
    4. Identify the new cluster centroids
    5. Repeat steps 3 and 4 until:
    	- Centroids no longer change
	        - Points remain in the same cluster
	        - Or max number of iterations is reached(defined by user)
	
	
![k-means example](https://github.com/kelseydang/BENG183_Project/blob/master/BENG183_Project/kmeans_example_figure.png)

from : https://www.edureka.co/blog/k-means-clustering/




Demo using K-means:
https://stanford.edu/class/ee103/visualizations/kmeans/kmeans.html

Optional exercise:
http://master.bioconductor.org/help/publications/books/bioinformatics-and-computational-biology-solutions/chapter-code/AnalClust.R

#### When to chose SVM or K-Means:

|**SVM**|**K-Means**|
|----|-----------|
|Data has labels, there are a lot of higher dimensions(categories such as genes) in the data|Data labels are unknown, a rough estimate of how many groups are present in data|
|Goal: Further classification of new data into its subtype|Goal: Cluster data to identify similar functions, identify subtypes, or discover shared similarity in DNA|

## 5\. Benefits of Precision Medicine

Precision medicine has improved the ability of doctors to predict which treatments would work best for which patients. This is because precision medicine allows practitioners and researchers to fully understand the mechanisms by which various disease pathways work. All of this culminates in an overall improved preventative, diagnosis, and treatment cycle. 
	
In a 2014 study involving 4,000 patients, Herceptin was shown to increase the survival rate of HER2-positive patients drastically when combined with chemotherapy. In fact, the 10-year survival rate increased from 75.2% to 84% when Herceptin was used additionally.<sup>26</sup>

## 6\. Drawbacks of Precision Medicine

Although the idea of predicting a treatment for each individual patient is ideal, there are many obstacles in Precision Medicine. The four main challenges presented in PM is: patient subtyping, biomarker discovery, drug repurposing and personalized treatment prediction.<sup>23</sup> With patient subtyping, the main task is to identify subpopulations of patients that can be used to guide treatment procedures of a given individual belonging to the subpopulation and to predict the outcomes²³. In subtyping there are endotypes and verotypes. Endotypes refer to the disease mechanisms similarities in patients and verotypes refer to populations of similar patients to predict treatments for the patients. The challenge with subtyping is that there are many different factors in subtyping patients, so incorporating all of the factors to group individuals would present difficulty in grouping very genetically different individuals.

Drug repurposing is defined as the, “identification and development of new uses for the existing or abandoned pharmacotherapies”.<sup>23</sup> There is an increasing abundance in the discovery of biomedical data such as biomarkers, the issue however, is finding a way to include and store this knowledge in large databases. In creating the databases, there is an issue with privacy in one’s health records being in these databases. Once the databases are all created, the next challenge is to extract the complementary information conveyed in data of different formats and types into an integrative method.<sup>23</sup> It would be difficult to find the correlation between the abundance of data taken from various types of omics data types. 

## 7\. Future Directions

Throughout this chapter we have solely focused on the omics data type of genomics, however there is an ever-growing amount of different omics data types being discovered today that can be incorporated into Precision Medicine. In addition to the abundance of omics data types, there is also a large number of clinical datasets of patient features becoming more available and more complex. In the future, the main focus would be to integrate these datasets and omics to create the most precise treatment for patients. In the figure below, it illustrates only some omics data types, even though there is much more to explore in the future.

![omics](https://github.com/kelseydang/BENG183_Project/blob/master/BENG183_Project/Multi_omics.png)

Some examples of different omics data types are: Transcriptomics, Proteomics and Interactomics, Phenomics and Exposomics. Transcriptomics measures the amount of transcribed genetic material over time. Proteomics and Interactomics focuses on produced proteins, after all translational modification. Phenomics measures the physical and biochemical traits of organisms, while Exposomics encompasses all human environmental, nongenetic, exposures after conception.<sup>23</sup> In the future, PM could not only be more precise and accurate with the growing knowledge of omics data types and the large clinical databases, but it would allow medical practitioners to correctly diagnose a patient and even utilize drug repurposing in giving treatment to patients as well. With the availability of various omics data, computational predictions of new drug candidates for repurposing has necessitated the development of many new methods for data integration, thus patients would be less misdiagnosed and have a greater success rate from the treatment when all of their health information is utilized in selecting treatment.<sup>23</sup>


## 8\. Conclusion

Today, the issue that is still presented is finding a way to intertwine
PM with other healthcare systems not just within the United States, but
also other countries that do not have the adequate knowledge nor
treatment for their patients. Ideally with Precision Medicine, it
changes the economic agenda and social policies of health not just
locally, but globally as well. Despite having a growing number of
genomic markers of efficacy and more discoveries of therapeutics for
clinical use, there still exists global barriers that prevent precision
medicine from making the impact that it has potential for. In order to
integrate PM technology into healthcare practice in the future there
needs to be changes within the insufficient decision supporting
technology and insufficient funding for translational health research.  

Perhaps in the future, economic and social policies will be created to
support the progress in precision medicine. This would be essential to
the adoption and integration of PM technologies into healthcare not just
in the United States, but worldwide. As biomarker discovery accelerates
and artificial intelligence-based technologies emerge, and with the help
of establishing health care policies that encourage precision medicine,
there is potential for the transition from disease treatment, to disease
prevention, and all the way to early detection.

## 9\. References
1. Precision medicine technology:https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5989714/
APA: Ginsburg, G. S., & Phillips, K. A. (2018). Precision Medicine: From Science To Value. Health affairs (Project Hope), 37(5), 694–701. doi:10.1377/hlthaff.2017.1624
2. Breast cancer milestones: https://www.healthline.com/health/history-of-breast-cancer#research-milestones
APA: 
3. Radical mastectomy:https://en.wikipedia.org/wiki/Radical_mastectomy
APA: Wikipedia contributors. (2019, November 23). Radical mastectomy. In Wikipedia, The Free Encyclopedia. Retrieved 06:45, December 9, 2019, from https://en.wikipedia.org/w/index.php?title=Radical_mastectomy&oldid=927529396
4. Xray breast cancer/radium radiation treatment: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5535674/
APA: Gianfaldoni, S., Gianfaldoni, R., Wollina, U., Lotti, J., Tchernev, G., & Lotti, T. (2017). An Overview on Radiotherapy: From Its History to Its Current Applications in Dermatology. Open access Macedonian journal of medical sciences, 5(4), 521–525. doi:10.3889/oamjms.2017.122
5. SERMs: https://www.breastcancer.org/treatment/hormonal/serms
APA:Selective Estrogen Receptor Modulators (SERMs). (2018, August 15). Retrieved from https://www.breastcancer.org/treatment/hormonal/serms.
6. Tamoxifen: https://www.breastcancer.org/treatment/hormonal/serms/tamoxifen
APA: Tamoxifen: Uses, Side Effects, and More. (2019, November 11). Retrieved from https://www.breastcancer.org/treatment/hormonal/serms/tamoxifen.
7. Herceptin: https://en.wikipedia.org/wiki/Trastuzumab
APA: Wikipedia contributors. (2019, December 3). Trastuzumab. In Wikipedia, The Free Encyclopedia. Retrieved 06:57, December 9, 2019, from https://en.wikipedia.org/w/index.php?title=Trastuzumab&oldid=929109857
8. Anastrozole: https://en.wikipedia.org/wiki/Anastrozole
APA: Wikipedia contributors. (2019, October 8). Anastrozole. In Wikipedia, The Free Encyclopedia. Retrieved 06:58, December 9, 2019, from https://en.wikipedia.org/w/index.php?title=Anastrozole&oldid=920183794
9. Raloxifene: https://en.wikipedia.org/wiki/Raloxifene
APA: Wikipedia contributors. (2019, November 25). Raloxifene. In Wikipedia, The Free Encyclopedia. Retrieved 06:58, December 9, 2019, from https://en.wikipedia.org/w/index.php?title=Raloxifene&oldid=927895736
10. Integrative methods for analyzing big data in precision medicine, https://onlinelibrary.wiley.com/doi/abs/10.1002/pmic.201500396
APA: Gligorijević, V., Malod‐Dognin, N. and Pržulj, N. (2016), Integrative methods for analyzing big data in precision medicine. Proteomics, 16: 741-758. doi:10.1002/pmic.201500396
11. BRCA1 and 2: https://www.cancer.gov/about-cancer/causes-prevention/genetics/brca-fact-sheet
APA: BRCA Mutations: Cancer Risk and Genetic Testing Fact Sheet. (2018, January 30). Retrieved from https://www.cancer.gov/about-cancer/causes-prevention/genetics/brca-fact-sheet.
12. More on PM and future directions https://www.ncbi.nlm.nih.gov/pubmed/30003435
APA: Love-Koh, J., Peel, A., Rejon-Parrilla, J. C., Ennis, K., Lovett, R., Manca, A., Chalkidou, A., et al. (2019). Correction to: The Future of Precision Medicine: Potential Impacts for Health Technology Assessment. PharmacoEconomics, 37(1), 117–117. Retrieved December 9, 2019, from 10.1007/s40273-018-0739-x
13. Other gene biomarkers associated with breast cancer: https://www.breastcancer.org/risk/factors/genetics
APA: (n.d.). Genetics: Breast Cancer Risk Factors. Breastcancer.org. Retrieved December 9, 2019, from https://www.breastcancer.org/risk/factors/genetics
14. First BRCA drug https://www.cancer.org/latest-news/fda-approves-first-drug-specifically-for-brca-mutated-breast-cancer.html
APA: (2018, January 22). FDA Approves First Drug Specifically For BRCA-mutated Breast Cancer. www.cancer.org. Retrieved December 9, 2019, from https://www.cancer.org/latest-news/fda-approves-first-drug-specifically-for-brca-mutated-breast-cancer.html
15. HER2:https://oncologypro.esmo.org/Education-Library/Factsheets-on-Biomarkers/HER2-in-Breast-Cancer
APA: Viale, G. (2015, August 5). HER2 In Breast Cancer: ESMO Biomarker Factsheet | OncologyPRO. OncologyPRO. Retrieved December 9, 2019, from https://oncologypro.esmo.org/Education-Library/Factsheets-on-Biomarkers/HER2-in-Breast-Cancer
16. BARD1https://www.facingourrisk.org/understanding-brca-and-hboc/information/hereditary-cancer/other-genes/basics/bard1.php
APA: (n.d.). BARD1 Mutation. Facingourrisk.org. Retrieved December 9, 2019a, from https://www.facingourrisk.org/understanding-brca-and-hboc/information/hereditary-cancer/other-genes/basics/bard1.php
17. BRIP1:https://mysupport360.com/associations/genes/brip1-gene-mutations/
APA: (2019, July 9). BRIP1 Gene Mutations - MySupport360. MySupport360. Retrieved December 9, 2019, from https://mysupport360.com/associations/genes/brip1-gene-mutations/
18. CDH1: https://mysupport360.com/associations/genes/cdh1-gene-mutations/
APA: (2019a, July 9). CDH1 Gene Mutations - MySupport360. MySupport360. Retrieved December 9, 2019, from https://mysupport360.com/associations/genes/cdh1-gene-mutations/
19. CHEK2:https://mysupport360.com/associations/genes/chek2-gene-mutations/
APA: (2019a, July 9). CHEK2 Gene Mutations - MySupport360. MySupport360. Retrieved December 9, 2019, from https://mysupport360.com/associations/genes/chek2-gene-mutations/
20. TP53:https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6460547/
APA: Lima, Z. S., Ghadamzadeh, M., Arashloo, F. T., Amjad, G., Ebadi, M. R., & Younesi, L. (2019). Recent advances of therapeutic targets based on the molecular signature in breast cancer: genetic mutations and implications for current treatment paradigms. Journal of hematology & oncology, 12(1), 38. doi:10.1186/s13045-019-0725-6
21. STK11:https://www.cancernetwork.com/cancer-and-genetics/targeting-inflammation-slows-stk11-mutation-driven-polyp-growth
APA: Furlow, B. (2018, August 14). Targeting Inflammation Slows STK11 Mutation-Driven Polyp Growth. Cancer Network. Retrieved December 9, 2019, from https://web.archive.org/web/20191209072648/https://www.cancernetwork.com/cancer-and-genetics/targeting-inflammation-slows-stk11-mutation-driven-polyp-growth
22. Support Vector Machines, an Overview, https://towardsdatascience.com/https-medium-com-pupalerushikesh-svm-f4b42800e989
APA: https://towardsdatascience.com/@pupalerushikesh. (2018a, June 16). Support Vector Machines(SVM) — An Overview. www.towardsdatascience.com. Retrieved December 9, 2019, from https://towardsdatascience.com/https-medium-com-pupalerushikesh-svm-f4b42800e989
23. “Integrative methods for analyzing big data in precision medicine”
https://docs.google.com/document/d/1_e2WeZ693Nq9DAGoDdQPsuzJJVXYxIailB96hJMLZl0/edit
APA: Gligorijević, V., Malod-Dognin, N., & Pržulj, N. (2016). Integrative methods for analyzing big data in precision medicine. Proteomics, 16(5), 741–758. Retrieved December 9, 2019, from 10.1002/pmic.201500396
24. Genomics and precision medicine https://www.sciencedirect.com/science/article/pii/B9780128006818000013
APA: Willard, H. F. (2017). The Human Genome, 1–19. Retrieved December 9, 2019, from 10.1016/b978-0-12-800681-8.00001-3
25. HER2 https://www.cancer.gov/research/progress/discovery/her2
APA: (2018b, April 11). HER2 Genetic Link To Breast Cancer. National Cancer Institute. Retrieved December 9, 2019, from https://www.cancer.gov/research/progress/discovery/her2
26. HER2 survival stats https://ascopubs.org/doi/10.1200/JCO.2014.55.5730
APA: Perez, E. A., Romond, E. H., Suman, V. J., Jeong, J.-H., Davidson, N. E., Geyer, C. E., Martino, S., et al. (2011). Four-Year Follow-Up of Trastuzumab Plus Adjuvant Chemotherapy for Operable Human Epidermal Growth Factor Receptor 2–Positive Breast Cancer: Joint Analysis of Data From NCCTG N9831 and NSABP B-31. JCO, 29(25), 3366–3373. Retrieved December 9, 2019, from 10.1200/jco.2011.35.0868

Images:
1. BRCA1 and 2:https://en.wikipedia.org/wiki/BRCA_mutation
APA:BRCA mutation. (2019, November 14). Retrieved from https://en.wikipedia.org/wiki/BRCA_mutation.
2. HER2https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(16)32417-5/fulltext
APA: Gianni, L. (n.d.). HER2-positive breast cancer. The Lancet. doi: https://doi.org/10.1016/S0140-6736(16)32417-5
3. Radical mastectomy: https://www.fairview.org/patient-education/83001
APA: Patient Education. (n.d.). Retrieved from https://www.fairview.org/patient-education/83001.
4. Radium radiation therapy: https://www.mskcc.org/timeline/msk-radiation-therapy-timeline-progress
APA:MSK Radiation Therapy: Timeline of Progress. (n.d.). Retrieved from https://www.mskcc.org/timeline/msk-radiation-therapy-timeline-progress.

## 10\. Supplementary Material

Additional breast cancer genomic biomarkers:

|Biomarker |Description |Patient type or Mutation result |Treatment |
|----------|------------|--------------------------------|----------|
|ATM |Gene that helps repair DNA. |Increased risk of breast cancer|Ongoing research|
|BARD1 |Works with BRCA1 to repair DNA |Increased risk of breast cancer|Ongoing research<sup>16</sup>|
|BRIP1|Gene that helps repair DNA.|Increased risk of breast cancer|Ongoing research<sup>17</sup>|
|CDH1|Protein that helps cells bind together to form tissue| invasive lobular breast cancer.|Ongoing research<sup>18</sup>|
|CHEK2|Gene that codes for tumor suppression protein.|More than double the risk of breast cancer.|Ongoing research<sup>19</sup>|
|MRE11A|Works with RAD50 and NBN genes, to form the MRN complex, which helps repair DNA.|Increases breast cancer risk|Ongoing research|
|MSH6|Codes for protein that repairs DNA.|Doubles the risk of breast cancer. |Ongoing research|
|NBN|Works with the MRE11A and RAD50 genes, to form MRN complex, which helps repair DNA. |Increases risk of breast cancer.|Ongoing research|
|PALB2|Works with BRCA2 to repair damaged DNA.|5-9x more likely to develop breast cancer.|Ongoing research.|
|PMS2|Codes for protein that repairs DNA.|Doubles breast cancer risk.|Ongoing research|
|PTEN|Regulates cell growth|Increases risk of breast cancer.| Ongoing research|
|RAD50|Works with the MRE11A and NBN genes, to form MRN complex, which helps repair DNA.|Increases risk of breast cancer.|Ongoing research|
|RAD51C|Repairs DNA damage.|Increases risk of breast cancer.|Ongoing research |
|STK11|Regulates cell growth.|Increases risk of breast cancer.|Undergoing research, one drug is metformin<sup>21</sup>|
|TP53|Gene that encodes tumor suppressor protein.|Nearly 100% chance of developing cancer. |Under research: APR-246 + azacitidine and Azacitidine20|

As you may see from the supplementary table above, many precision treatments do not exist for the biomarkers associated with breast cancer. Some drugs are currently undergoing studies while others are yet to be discovered. 


Additional Different “-Omics” Table:

|NAME|Type of Data|
|----|-----------|
|Genomics|Whole Genomes (DNA)|
|Exomics|Smaller Regions of genomes; Exons|
|Epigenomics|Reversible modifications of DNA/histones, DNA methylation, Chromatin Structure|
|Transcriptomics|Coding and non-coding RNAs, transcribed material|
|Proteomics|Produced proteins post-translational modifications|
|Interactomics|Physical or chemical interactions|
|Metabolomics|All chemical processes involving metabolites|
|Glycomics|Branch of metabolics: focuses on sets of all sugars either free or in complex molecules in cells|
|Fluxomics|Branch of metabolics that attempts to predict rates of metabolic reactions in biological systems|
|Phenomics|Measurement of Physical or biochemical traits of organisms as they change in response to genetic mutation and environmental influences|
|Exposomics|Encompasses exposures such as toxic molecules, drugs, radiation, etc.|
|Metagenomics|Aims at capturing human microbiomes. i.e. through 16s rRNA sequencing|

